diff --git a/scripts/configure.sh b/scripts/configure.sh
index 593949b..0ed0c4c 100644
--- a/scripts/configure.sh
+++ b/scripts/configure.sh
@@ -2,9 +2,10 @@
 
 set -o errexit -o pipefail -o noclobber -o nounset
 
-cmake . -G Ninja -B build \
+cmake . -G "Unix Makefiles" -B build \
     -DSYCL_IMPLEMENTATION=ComputeCpp \
     -DCMAKE_PREFIX_PATH=/sycl \
+    -DSYCL_LANGUAGE_VERSION=2020 \
     -DCMAKE_BUILD_TYPE=Release \
     -DSYCL_CTS_ENABLE_FULL_CONFORMANCE=0 \
     $@
